core/rawdb/accessors_trie.go

```diff
@@ -141,6 +141,24 @@ func DeleteStorageTrieNode(db ethdb.KeyValueWriter, accountHash common.Hash, pat
 	}
 }
 
+// HasTrieNodeInPath checks for the presence of the trie node with the specified
+// account hash and node path, regardless of the node hash.
+func HasTrieNodeInPath(db ethdb.KeyValueReader, accountHash common.Hash, path []byte) bool {
```

Comment:
Since for all other operation we have separate methods for Account and Storage TrieNode, I'd recommend making this also separate. HasAccountTrieNodeInPath and HadStorageTrieNodeInPath. Also, do we need the InPath suffix? Isn't that implicit?

Reply:
Yeah, we already defined `HasAccountTrieNode(db ethdb.KeyValueReader, path []byte, hash common.Hash)`, so..

Reply:
That is basically the same thing, just does a hash check too? Ok, not the same, because it does a Get vs HAs doesn't need to touch the value tables.

Reply:
Could we then have?

ExistsAccountTrieNode
ExistsStorageTrieNode